common/icons/information.js

```diff
@@ -1,5 +1,5 @@
 const information = () => (
-  <svg data-name="Layer 1" viewBox="0 0 26 26"><path className="cls-1" d="M15.25 19.34H14v-7.27a1 1 0 0 0-1-1h-2.25a1 1 0 0 0 0 2H12v6.26h-1.25a1 1 0 0 0 0 2h4.5a1 1 0 1 0 0-2z"/><circle className="cls-1" cx="13" cy="6.79" r="2.13"/></svg>
+  <svg data-name='Layer 1' viewBox='0 0 26 26'><path className='cls-1' d='M15.25 19.34H14v-7.27a1 1 0 0 0-1-1h-2.25a1 1 0 0 0 0 2H12v6.26h-1.25a1 1 0 0 0 0 2h4.5a1 1 0 1 0 0-2z'/><circle className='cls-1' cx='13' cy='6.79' r='2.13'/></svg>
```

Comment:
We can/should get rid of `data-name` and `cls-x` classes (I also think we should have `icon__shape` on every svg part that we want to be able to change the colour of) – maybe one for another PR.

Reply:
How do we reckon we can do this using `svgo`?